app/views/handlers/list_collector.py

```diff
@@ -65,18 +64,21 @@ def get_context(self) -> dict[str, dict]:
             self._questionnaire_store.supplementary_data_store,
         )
 
-        return {
-            **question_context,
-            **list_context(
-                self.rendered_block["summary"],
-                for_list=self.list_name,
-                edit_block_id=self.rendered_block["edit_block"]["id"],
-                remove_block_id=self.rendered_block["remove_block"]["id"],
-                return_to=self._return_to,
-                section_id=self.current_location.section_id,
-                has_repeating_blocks=bool(self.repeating_block_ids),
-            ),
-        }
+        return list_context(
+            self.rendered_block["summary"],
+            for_list=self.list_name,
+            edit_block_id=self.rendered_block.get("edit_block", {}).get("id"),
+            remove_block_id=self.rendered_block.get("remove_block", {}).get("id"),
+            return_to=self._return_to,
+            section_id=self.current_location.section_id,
+            has_repeating_blocks=bool(self.repeating_block_ids),
+        )
+
+    def _get_additional_view_context(self) -> dict:
+        return super().get_context()
```

Comment:
I'd find it helpful to have a docstring on this one, mentioning what it does in the case of a list collector - initially I wasn't sure why this was added instead of using the super method in `get_context` but now I see its so it can be overwritten in list collector content